controllers/mongodbcluster_controller.go

```diff
@@ -472,3 +528,159 @@ func (r *MongoDBClusterReconciler) reconcileAtlasFirewall(ctx context.Context, m
+		if err != nil {
+			logger.Error(err, "Couldn't find cluster in Atlas")
+			return err
+		}
+
+		clusterDetails, response, err := client.Clusters.Get(ctx, atlasGroupID, clusterName)
+		if err != nil || response.StatusCode != http.StatusOK {
+			logger.Error(err, "Couldn't get cluster details from Atlas")
+			return err
+		}
+
+		if foundDown == nil || foundDown.Tags()[1] != scheduledAutoscaling.ScaleDownCronExpression+scheduledAutoscaling.LowTier {
+			if foundDown != nil {
+				logger.Info("Removing outdated downscaling job for " + mongodbClusterCR.Name)
+
+				err = r.Scheduler.RemoveJob(foundDown.ID())
+				if err != nil {
+					logger.Error(err, "Error removing outdated downscaling job")
+					return err
+				}
+			}
+
+			logger.Info("Creating scheduled downscaling job for " + mongodbClusterCR.Name + " with expression " + scheduledAutoscaling.ScaleDownCronExpression + " to " + scheduledAutoscaling.LowTier)
+
+			_, err = r.Scheduler.NewJob(
+				gocron.CronJob(scheduledAutoscaling.ScaleDownCronExpression, false),
+				gocron.NewTask(
+					func() error {
+						logger.Info("Scaling down " + mongodbClusterCR.Name + " to " + scheduledAutoscaling.LowTier)
+
+						r.EventRecorder.Event(mongodbClusterCR, corev1.EventTypeNormal, "Scaling", "Scaling down to "+scheduledAutoscaling.LowTier)
+
+						_, response, err := client.Clusters.Update(ctx, atlasGroupID, clusterName, &mongodbatlas.Cluster{
+							ProviderSettings: &mongodbatlas.ProviderSettings{
+								ProviderName:     "AWS",
+								InstanceSizeName: scheduledAutoscaling.LowTier,
+								RegionName:       clusterDetails.ProviderSettings.RegionName,
+							},
+						})
+
+						if err != nil || response.StatusCode != http.StatusOK {
+							logger.Error(err, "Couldn't scale down "+mongodbClusterCR.Name)
+							r.EventRecorder.Event(mongodbClusterCR, corev1.EventTypeWarning, "Scaling", "Failed to scale down to "+scheduledAutoscaling.LowTier)
+
+							// Flip a metric so we can alert on this. This one is a warning.
+							ScalingDownErrorGauge.WithLabelValues(mongodbClusterCR.Name).Inc()
+
+							return err
+						}
+
+						// Reset metric if the reconcile then succeeded
+						ScalingUpErrorGauge.WithLabelValues(mongodbClusterCR.Name).Set(0)
+
+						return nil
+					},
+				),
+				gocron.WithTags(mongodbClusterCR.Name, scheduledAutoscaling.ScaleDownCronExpression+scheduledAutoscaling.LowTier, "down"),
+			)
+
+			if err != nil {
+				logger.Error(err, "Error creating new upscaling job")
+				return err
+			}
+		}
+
+		if foundUp == nil || foundUp.Tags()[1] != scheduledAutoscaling.ScaleUpCronExpression+scheduledAutoscaling.HighTier {
+			if foundUp != nil {
+				logger.Info("Removing outdated upscaling job for " + mongodbClusterCR.Name)
+
+				err = r.Scheduler.RemoveJob(foundUp.ID())
+				if err != nil {
+					logger.Error(err, "Error removing outdated upscaling job")
+					return err
+				}
+			}
```

Comment:
```suggestion
		if foundUp != nil && foundUp.Tags()[1] != scheduledAutoscaling.ScaleUpCronExpression+scheduledAutoscaling.HighTier {
			logger.Info("Removing outdated upscaling job for " + mongodbClusterCR.Name)

			err = r.Scheduler.RemoveJob(foundUp.ID())
			if err != nil {
				logger.Error(err, "Error removing outdated upscaling job")
				return err
			}
			
			foundUp = nil;
		}
		
		if foundUp == nil {
			// maybe call a func...
```

Can do similar for foundDown section